integrations/spark/spark-job-dataquality/spark-job-dataquality-app/src/test/java/com/thinkbiganalytics/spark/dataquality/rule/RowCountRuleTest.java

```diff
@@ -0,0 +1,46 @@
+package com.thinkbiganalytics.spark.dataquality.rule;
```

Comment:
Nice to add a test that detects failure when source count != valid+invalid counts